### ISSUE TITLE
Clone repository for Observation Framework

### ISSUE THREAD

**Issue by gitwjr** [cta-wave/dpctf-deploy#99]
@louaybassbouss @yanj-github 
It appears the Deploy readme has the wrong link for cloning the device-observation-framework repository.
Clone repository
Using the git command line tool, you can download the current version of the dpctf-deploy repository to your system:

Linux / macOS / WSL2:

git clone https://github.com/cta-wave/dpctf-deploy

Shouldn't it be git clone https://github.com/cta-wave/device-observation-framework ?
I tried git clone https://github.com/cta-wave/dpctf-deploy and it just made another copy of the TR within dpctf-deploy and no observation framework folder was created. When I changed it to ...cta-wave/devoce-observation-framework it created the folder and its contents. 

Am I correct?

**Comment by yanj-github** [comment]
I think this is correct as ./build-dof.sh or .\build-dof.bat will clone the https://github.com/cta-wave/device-observation-framework.git.
If you already cloned https://github.com/cta-wave/dpctf-deploy before and wants to set up OF same PC/sever just need to skip the cloning step.

**Comment by gitwjr** [comment]
After building (and docker-compose up) the TR, the device-observation=framework folder (and its contents) did not show up in DPCTF-Deploy folder. It was not until I cloned the Device OF and built it that it appeared and I could run build-dof.sh.

**Comment by gitwjr** [comment]
After discussions with Louay and Fritz it appears that there is no need to re-clone the repository as stated in Phase 3 of the Docker README instructions if you have already cloned and built the TR repository in Phase 2. The necessary files are already included in dpctf-deploy folder to run docker-compose up. This should be made clear in the instructions. Otherwise it creates a new dpctf-deploy folder within the already existing dpctf-deploy folder that was created when building the TR which is confusing, especially since the new folder will not have the imported content and some people may change the wrong config.json and docker-compose.yml files.

**Comment by FritzHeiden** [comment]
added note to skip cloning repository if already done with https://github.com/cta-wave/dpctf-deploy/pull/105

**Comment by gitwjr** [comment]
The change addresses the issue. Close.
